lib/Dist/Zilla/Plugin/Test/Perl/Critic.pm

```diff
@@ -116,10 +131,5 @@ ___[ xt/author/critic.t ]___
 use strict;
 use warnings;
 
-use Test::More;
-use English qw(-no_match_vars);
```

Comment:
The removal of this one line is sufficient grounds to release this PR immediately -- now that Dist::Zilla::Plugin::AutoPrereqs now scans xt/ for develop-requires prereqs, English is showing up as a develop prereq in many distributions.